manifest.php

```diff
@@ -63,9 +65,10 @@
     'update' => Updater::class,
     'managementRole' => 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LtiManagerRole',
     'acl' => [
-        ['grant', 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LtiManagerRole', ['ext' => 'taoLti']],
-        ['grant', TaoRoles::ANONYMOUS, CookieUtils::class],
-        ['grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BaseUserRole', ['ext' => 'taoLti','mod' => 'LtiConsumer', 'act' => 'call']]
+        [AccessRule::GRANT, 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LtiManagerRole', ['ext' => 'taoLti']],
```

Comment:
i would turn
`http://www.tao.lu/Ontologies/TAOLTI.rdf#LtiManagerRole` into a constant

Reply:
I will in the next PR where I have to change tao-core too :)